UI Customization document - initial chapters (#35579)
keycloak/keycloak: docs/documentation/ui_customization/topics/welcome-theme.adoc

[[_welcome_theme]]
== Customizing the Welcome Theme

> The welcome theme is simply the web page that is served when you request the default page from the {project_name} server.  For instance, if your server is deployed on your local machine at port 8080 then http://localhost:8080 will serve the welcome theme.

Reviewer: ```suggestion
The welcome theme is the web page that is served when you request the default page from the {project_name} server.  For instance, if your server is deployed on your local machine at port 8080, http://localhost:8080 serves the welcome theme.
```

> By default, the welcome theme is only used to create the initial, temporary admin user.  Once that user is created, any time users navigate to the welcome theme, they will be redirected to the admin console.  However, this behavior can be changed and the welcome theme can be completely customized or replaced.

Reviewer: ```suggestion
By default, the welcome theme is only used to create the initial temporary admin user.  Once that user is created, whenever users navigate to the welcome theme, they are redirected to the Admin Console.  However, this behavior can be changed and the welcome theme can be completely customized or replaced.
```

> Since the welcome theme is not associated with a realm, it can not be selected in the admin console like other themes.

Reviewer: ```suggestion
Since the welcome theme is not associated with a realm, it cannot be selected in the admin console like other themes.
```

> To change the welcome theme, simply create and deploy a new welcome theme as discussed in <<_creating-a-theme,Creating a theme>>.  Then, start the keycloak server using the `spi-theme-welcom-theme` option.

Reviewer: ```suggestion
To change the welcome theme, create and deploy a new welcome theme as described in <<_creating-a-theme,Creating a theme>>.  Then, start the {project_name} server using the `spi-theme-welcom-theme` option.
```
